packages/components/data-table/src/data-table.styles.js

```diff
@@ -28,6 +28,7 @@ const getDisabledSelfContainmentStyles = (props) => {
 };
 
 const TableContainer = styled.div`
+  width: fix-content;
```

Comment:
wow, i didn't know that css property exists

Reply:
I think it's relatively new (among with `max-content` and `min-content`), but it's supported by all major browsers 🙂 

Really handy for this use-case.